### PR TITLE
First pass at a PR template

### LINKED ISSUE #1: Create a default pull request template
### Is your feature request related to a problem? Please describe.

We have some GitHub Guidelines now, which include guidance around branch naming, writing issues, referencing issues from pull requests etc.

### Describe the solution you’d like

A pull request template would help serve as a reminder of these guidelines. It’s not perfect (e.g. I’m not sure it would be visible if the PR were created via a client like [gh](https://github.com/cli/cli)) but it’s probably the most useful place to put this information, where it’s most likely to be seen.

### PR DESCRIPTION
**Pull request by andylolz**
Fixes #1.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+The more detailed you are, the better.**
+
+-----------------
+
+## Pull request checklist
+
+- [ ] I have linked my PR to an issue
+- [ ] I am content that my [commit messages are good](https://chris.beams.io/posts/git-commit/)
+- [ ] My branch is up-to-date with `main`
+- [ ] Where appropriate, I have added tests
```
Comment (JamesMcMinn) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: "added or updated tests"?

**Review by JamesMcMinn — APPROVED** [review]
This is good - a better than than the nothing we have right now!

I wonder if it would also be useful to include a `closes #X`/`fixes #X` somewhere in the body so that it's actually more of a pain to remove it than it is to just put in the correct issue number? 

Regardless, I think this would be good to go in its current form.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+**Replace this text with a summary of the changes in your PR.
+The more detailed you are, the better.**
+
+-----------------
+
+## Pull request checklist
+
+- [ ] I have linked my PR to an issue
+- [ ] I am content that my [commit messages are good](https://chris.beams.io/posts/git-commit/)
+- [ ] My branch is up-to-date with `main`
```
[the remark this answers is not in the document]
Comment (andylolz) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: We can and we should (and on live-api and audio-transcriber, we do). I’d still leave this checkbox in as a reminder. But I’d also be fine with taking it out.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (andylolz) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Good point

**Review by JamesMcMinn — APPROVED** [review]
Looks good to me!

**Pull request merged**